### PR TITLE
chore: update waku deps, add notes example, move other examples to experimental

### LINKED ISSUE #59: chore: move existing examples under experimental and add notes example
Moving forward, we want to focus on maintaining a single flagship example in this repository that is updated with every release of js-waku packages. This will be the notes example from this PR: https://github.com/waku-org/examples.waku.org/pull/286

The other examples should be moved to an `experimental` folder; we want to retain the code for future reference, as we may implement some of the same features in the notes example or re-introduce them as one of the examples we maintain.

### PR DESCRIPTION
**Pull request by adklempner**
- Remove relay chat examples
- Move all existing examples under `experimental`
- Add notes example from https://github.com/waku-org/examples.waku.org/pull/286
- Update notes example to work with latest/rc waku packages

Resolves #59

### CODE REVIEW

**Review by danisharora099 — APPROVED** [review]
@adklempner approving since work has been done here but I'd propose to remove the examples we don't require to maintain cc @weboko 

also some CIs fail

File: examples/experimental/light-js/index.html [hunk 1]
```diff
@@ -76,11 +76,11 @@ <h2>Store</h2>
         createDecoder,
         utf8ToBytes,
         bytesToUtf8,
-      } from "https://unpkg.com/@waku/sdk@0.0.24-070b625.0/bundle/index.js";
+      } from "https://unpkg.com/@waku/sdk@0.0.24/bundle/index.js";
       import {
         enrTree,
         DnsNodeDiscovery,
-      } from "https://unpkg.com/@waku/discovery@0.0.2-070b625.0/bundle/index.js";
```
Comment (weboko) [inline examples/experimental/light-js/index.html:83]: is it intentional? I think latest is better to be used

File: examples/experimental/noise-js/package.json [hunk 2]
```diff
@@ -9,9 +9,9 @@
     "start": "webpack-dev-server"
   },
   "dependencies": {
-    "@waku/sdk": "0.0.18",
-    "@waku/noise": "0.0.3-31510da",
```
Comment (weboko) [inline examples/experimental/noise-js/package.json:13]: please, check which one is latest version of noise

File: examples/experimental/light-js/index.html (hunk 1 above)
Comment (adklempner) [reply examples/experimental/light-js/index.html]: updated to use current next tag

File: examples/experimental/noise-js/package.json (hunk 2 above)
Comment (adklempner) [reply examples/experimental/noise-js/package.json]: updated to use current next tag

**Pull request merged**